Core/Object Arts/Dolphin/Base/Object.cls

```diff
@@ -1319,11 +1319,61 @@ stbSaveOn: anSTBOutFiler
 
 	anSTBOutFiler saveObject: self!
 
+storeAttributesDifferingFrom: default On: aStream variables: aDictionary indent: anInteger
+
+	| aspects rejects specialAspects |
+	rejects := #(#'controlDispatch' #'yourself').
+	specialAspects := IdentityDictionary new
+		at: #'commandDescription' 	put: [:anObject | aStream nextPutAll: 'command: '; print: anObject command];
+		at: #'buddy'			 	put: [:anObject | 
+			self isAutoBuddy ifTrue: [
+				aStream nextPutAll: 'isAutoBuddy: true'
+			] ifFalse: [
+				aStream nextPutAll: 'buddy: '; nextPutAll: (aDictionary at: anObject)
+			]
+		];
+		yourself.
+	aspects := self class publishedAspectsOfInstances asSortedCollection.
```

Comment:
#publishedAspects is a development system package concept. You simply can't use this in the base. This method assumes too much knowledge of specifics of particular classes anyway to be appropriate as an Object method.